Apex_Trigger/CustomStatus.trigger

```diff
@@ -0,0 +1,26 @@
+trigger CustomStatus on Class__c (before update,after update) 
+{
+      if(Trigger.isBefore)
+      {
+        for(Class__c cls : Trigger.new)
+        {
+            if(cls.Custom_Status__c == 'Reset')
+            {
+                cls.MyCount__c = 0;
+            }
+        }
+      }
+       else
+       {  
+            Set<Id> classSet = new Set<Id>();
+            for(Class__c cls : Trigger.new)
+            {
+                if(cls.Custom_Status__c == 'Reset')
```

Comment:
it's just checking if the value is `Reset` it's not checking if it's changed

Reply:
Done it.